bindings/python/examples/exchange/5_send_amount.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from iota_sdk import Wallet, SyncOptions
+from dotenv import load_dotenv
+import os
+
+# This example uses secrets in environment variables for simplicity which should not be done in production.
+load_dotenv()
+
+# This example sends tokens to an address.
```

Comment:
Why isn't the description of all(?) of the python examples not at the top right below the Copyright? :thinking: IMHO this should be the first thing the user should read to get behind the idea of each example.